documentation/docs/install-pmm/install-pmm-server/deployment-options/helm/index.md

```diff
@@ -82,33 +132,106 @@ Create the required Kubernetes secret and deploy PMM Server using Helm:
         percona/pmm 1.3.21          2.44.0      A Helm chart for Percona Monitoring and Managem...
         ```
 
-5. Deploy PMM Server with your chosen version and secret:
+6. Deploy PMM Server with your chosen version and secret:
 
+=== "For Kubernetes"
     ```bash
-    # Choose a specific chart version from the list in previous step
     helm install pmm \
     --set secret.create=false \
     --set secret.name=pmm-secret \
     --version 1.4.3 \
     percona/pmm
     ```
 
-6. Verify the deployment:
+=== "For OpenShift"
+    
+    - Create a custom values file for OpenShift:
+      ```bash
+      cat <<EOF > openshift-values.yaml
+      secret:
+        create: false
+        name: pmm-secret
+      
+      # OpenShift-specific pod security settings
+      podSecurityContext:
+        runAsNonRoot: true
+        runAsGroup: 1000
+        seccompProfile:
+          type: RuntimeDefault
+      EOF
+      ```
+    
+    - Deploy using the values file:
+      ```bash
+      helm install pmm \
+      -f openshift-values.yaml \
+      --version 1.4.3 \
```

Comment:
There is no 1.4.3 for Openshift. @BupycHuk which version will be correct for OpenShift?

Reply:
1.4.8